client/src/modules/invoices/registry/search.modal.html

```diff
@@ -50,22 +50,17 @@
           </div>
         </div>
 
-        <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.service.$invalid }">
-          <label class="control-label" translate>FORM.LABELS.SERVICE</label>
+        <bh-service-select
+          service-id="$ctrl.searchQueries.service_id"
+          name="service_id"
```

Comment:
It doesn't look like you define a "name" binding.  You probably can just remove this.